docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -57,3 +57,58 @@ Result:
 │       6 │
 └─────────┘
 ```
+
+## seriesDecomposeSTL
+
+Decompose time series data based on STL(Seasonal-Trend Decomposition Procedure Based on Loess)
+Returns an array of three arrays where the first array include seasonal components, the second array - trend,
```

Comment:
The specifics of what is returned (l. 64-65) can be moved to the `**Returned value**` sub-section below.